src/commons_codec/model.py

```diff
@@ -11,28 +12,25 @@
 else:
     from backports.strenum import StrEnum  # pragma: no cover
 
+try:
+    from sqlalchemy_cratedb.support import quote_relation_name
+except ImportError:  # pragma: no cover
+    quote_relation_name = None
```

Comment:
Is this a good pattern as without quoting it can lead to unexpected behaviour (wrong names used, maybe even deleting wrong data/tables)?

Reply:
Yes I think we must depend on it, unexpected behaviour is worse than this dependency ;)

Reply:
Let's do it. Thanks!

Reply:
I've amended the patch according to you suggestions, it is much more straight-forward now. Thank you!